shell/src/main/java/alluxio/cli/fs/command/DecommissionWorkerCommand.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fs.command;
+
+import alluxio.Constants;
+import alluxio.cli.fs.FileSystemShellUtils;
+import alluxio.client.block.BlockMasterClient;
+import alluxio.client.block.BlockWorkerInfo;
+import alluxio.client.file.FileSystemContext;
+import alluxio.exception.AlluxioException;
+import alluxio.grpc.DecommissionWorkerPOptions;
+import alluxio.resource.CloseableResource;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Decommission a specific worker, the decommissioned worker is not automatically
+ * shutdown and are not chosen for writing new replicas.
+ */
+public final class DecommissionWorkerCommand extends AbstractFileSystemCommand {
+
+  private static final int DEFAULT_TIMEOUT = 10 * Constants.MINUTE_MS;
+  private static final Option TIMEOUT_OPTION =
+      Option.builder("t")
+          .longOpt("timeout")
+          .argName("timeout in seconds")
+          .numberOfArgs(1)
+          .desc("Timeout in milliseconds for decommissioning a"
+              + "single worker, default: " + DEFAULT_TIMEOUT)
+          .required(false)
+          .build();
```

Comment:
on 2nd thought, i think this timeout is not very necessary. The master receives the call and just removes the block locations. This is not reversible so it doesn't really matter if you timeout on the client side or not. Even if your shell is dead, the worker will still be decommissioned.

let's remove this option then